src/ripple/app/tx/impl/SusPay.cpp

```diff
@@ -103,8 +101,10 @@ namespace ripple {
         If the SusPay ledger entry specifies sfCancelAfter, the
         transaction will fail if sfCancelAfter <= parentCloseTime.
 
-        NOTE: It must always be possible to verify the condition
-              without retrieving the SusPay ledger entry.
+        NOTE: The reason the condition must be specified again
+              is because it must always be possible to verify
+              the condition without retrieving the SusPay
+              ledger entry.
```

Comment:
I keep thinking we have the Fulfillment _object_ available in `doApply`. We don't. Deriving the condition from the deserialized fulfillment inside of `doApply` is a bad idea. Yeah, I agree, let's send the condition with the tx.